### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by natiki**
Added instructions for Windows users. Made use of the inbuilt certutil command to do the base64 decoding.

### CODE REVIEW

**Comment by prydonius** [comment]
@sameersbn would you have some time to test out these instructions on Windows?

File: docs/user/getting-started.md [hunk 1]
```diff
@@ -24,11 +24,33 @@ The above commands will deploy Kubeapps into the `kubeapps` namespace in your cl
 
 Access to the Dashboard requires a Kubernetes API token to authenticate with the Kubernetes API server.
 
-```
+For Linux:
+
+```bash
```
Comment (sameersbn) [inline docs/user/getting-started.md]: You need to move the following two commands outside the platform specific instructions right?

```bash
kubectl create serviceaccount kubeapps-operator
kubectl create clusterrolebinding kubeapps-operator --clusterrole=cluster-admin --serviceaccount=default:kubeapps-operator 
```

File: docs/user/getting-started.md [hunk 2]
```diff
@@ -24,11 +24,33 @@ The above commands will deploy Kubeapps into the `kubeapps` namespace in your cl
...
 kubectl create serviceaccount kubeapps-operator
 kubectl create clusterrolebinding kubeapps-operator --clusterrole=cluster-admin --serviceaccount=default:kubeapps-operator
 kubectl get secret $(kubectl get serviceaccount kubeapps-operator -o jsonpath='{.secrets[].name}') -o jsonpath='{.data.token}' | base64 --decode
 ```
+For Windows:
+
+Create a file called GetDashToken.cmd with the following lines in it:
```
Comment (sameersbn) [inline docs/user/getting-started.md]: Please format instances of GetDashToken.cmd as `GetDashToken.cmd`

File: docs/user/getting-started.md [hunk 3]
```diff
@@ -24,11 +24,33 @@ The above commands will deploy Kubeapps into the `kubeapps` namespace in your cl
...
+SET /p ks=<s.txt
+DEL s.txt
+
+REM Get the Base64 encoded token
+kubectl get secret %ks% -o jsonpath={.data.token} > b64.txt
+
+REM Decode The Token
+DEL token.txt
+certutil -decode b64.txt token.txt
+
```
Comment (sameersbn) [inline docs/user/getting-started.md]: please remove this extra new-line character

File: docs/user/getting-started.md [hunk 4]
```diff
@@ -24,11 +24,33 @@ The above commands will deploy Kubeapps into the `kubeapps` namespace in your cl
...
+```
+Open a command prompt and run the GetDashToken.cmd Your token can be found in the token.txt file.
```
Comment (sameersbn) [inline docs/user/getting-started.md]: Please add the missing period `.`

File: docs/user/getting-started.md [hunk 5]
```diff
@@ -24,11 +24,33 @@ The above commands will deploy Kubeapps into the `kubeapps` namespace in your cl
...
+```bash
 kubectl create serviceaccount kubeapps-operator
 kubectl create clusterrolebinding kubeapps-operator --clusterrole=cluster-admin --serviceaccount=default:kubeapps-operator
 kubectl get secret $(kubectl get serviceaccount kubeapps-operator -o jsonpath='{.secrets[].name}') -o jsonpath='{.data.token}' | base64 --decode
 ```
+For Windows:
+
+Create a file called GetDashToken.cmd with the following lines in it:
+
+```
```
Comment (sameersbn) [inline docs/user/getting-started.md]: You can use `bat` to apply syntax highlighting to this block

f.e. 

```bat
REM my comment
DEL s.txt
```

**Comment by sameersbn** [comment]
ping @natiki

**Comment by natiki** [comment]
ping @sameersbn

File: docs/user/getting-started.md (hunk 3 above)
Comment (prydonius) [reply docs/user/getting-started.md]: @natiki I think you still need to remove this?

Comment (prydonius) [reply docs/user/getting-started.md]: ah I saw your comment about readability, we can leave it in

**Pull request merged**